tests/integration/handle.py

```diff
@@ -21,6 +22,8 @@ def handle(event, context):
 
     # Generate custom metrics
     lambda_metric("hello.dog", 1, tags=["team:serverless", "role:hello"])
+    lambda_metric("hello.cat", 1, tags=["team:serverless", "role:hello"],
+        timestamp=time.time() - 60)
```

Comment:
Creating a custom metric with a timestamp forces our instrumentation to use the `datadog` api package instead of sending the metric to the extension.